.prettierrc.json

```diff
@@ -0,0 +1,4 @@
+{
+    "tabWidth": 4,
+    "singleQuote": true
+}
```

Comment:
i also use a line width of 100. forces you to keep the code structure simple/flat and helps with diffs / pull requests.

Reply:
done. added print width of 100